statsmodels/discrete/tests/results/results_discrete.py

```diff
@@ -918,7 +918,7 @@ def negativebinomial_nb1_bfgs(self):
                         # from stata for alpha no lnalpha
                          [ 3.609675,    3.856716]]
         #   [  1.28360034e+00,   1.34979803e+00]]
-        self.fittedvalues = [ 0.8487497 ,  0.8487497 ,  0.8487497 ,  0.8487497,
+        self.linpred = [ 0.8487497 ,  0.8487497 ,  0.8487497 ,  0.8487497,
```

Comment:
Since this is getting edited anyway, can you pls document-or-delete the commented-out stuff on line 920?